### DISCUSSION TITLE
Quieter Beacon Sounds

### DISCUSSION THREAD

**Discussion by gibsoniguess** [BedrockTweaks/Files#233] [Ideas]
I have 5 beacons in my base and they are so loud, it makes it unbearable sometimes.

**Comment by SquatchHunter** [comment]
ill do another pass on the peace and quiet packs to grab some of the missing ones, NSPC did them initially

update: pack is made, waiting on approval from DrAv before I open a issue & PR to merge it in, since its not a "confirmed" pack, I need approval before going ahead and requesting it be merged.

**Comment by SquatchHunter** [comment]
@drav0011 see above.
